packages/components/comment/style/themes/default.variable.less

```
@@ -0,0 +1,15 @@
+@comment-padding-md: @spacing-md 0;
+@comment-margin-right: @margin-md;
+@comment-font-size-md: @form-font-size-md;
+@comment-font-size-sm: @form-font-size-sm;
```

```suggestion
@comment-margin-right: @spacing-md;
@comment-font-size-md: @font-size-md;
@comment-font-size-sm: @font-size-sm;
```

comment 组件不应该依赖 form 的 less 变量。